dev/managedservicesplatform/spec/monitoring.go

```diff
@@ -9,6 +10,14 @@ import (
 )
 
 var codeClassPattern = regexp.MustCompile(`\dx+`)
+var customAlertNamePattern = regexp.MustCompile(`^[-A-Za-z0-9 ]+$`)
+
+type SeverityLevel string
+
+const (
+	SeverityLevelWarning  = "WARNING"
+	SeverityLevelCritical = "CRITICAL"
+)
```

Comment:
```suggestion
type SeverityLevel string

const (
	SeverityLevelWarning  SeverityLevel = "WARNING"
	SeverityLevelCritical SeverityLevel = "CRITICAL"
)
```

Otherwise defining the type `SeverityLevel` is ineffective... 😂

Reply:
ah, good catch
I moved this from https://github.com/sourcegraph/sourcegraph/pull/62885/files#diff-5c13ac2fa60a95f430297d8771799b213fa23c02dfcc719ea54a955097d47360L108-L113 without too much thought and never noticed it wasn't using the types already :man_facepalming: